deebotozmo/commands.py

```diff
@@ -29,6 +29,23 @@ def __init__(self, speed: str):
         })
 
 
+class SetScheduleOnce(Command):
+
+    def __init__(self, hour: str, minute: str ):
+        super().__init__("setSched", {
+            "act": "add",
+            "content": {"jsonStr":"{\"type\":\"auto\"}","name":"clean"},
+            "enable": 1,
+            "hour": int(hour),
+            "index": 2,
```

Comment:
What does the index? Must we set a correct value?

Reply:
After some test`s i'm not 100% sure. Maybe we must raise the value after we read the schedule list...